script.js

```diff
@@ -0,0 +1,15 @@
+const produto = {
+    item: "Notebook",
+    preco: 5.500,
+    marca: "Dell",
+    categoria: "Eletrônico",
+    descricao: "Windows, 16Gb RAM, 512Gb SSD"
+  };
+
+  const exibirElement = document.querySelector('#exibir');
```

Comment:
muito bom mas acho poderia ser outro nome de variavel exemplo abaixo:

```suggestion
  const exibirComponente = document.querySelector('#exibir');
```